### PR TITLE
Update IAP wording

### PR DESCRIPTION
**Pull request by sunshinejr**

### CODE REVIEW

File: Gifski/InternetAccessPolicy.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
 			"NetworkProtocol" : "TCP",
 			"Port" : "443",
 			"Purpose" : "Gifski sends crash reports to this server.",
-			"DenyConsequences" : "If you deny this connection, the developers of Gifski won't be notified of any crashes and they can't fix what they don't know about."
```
Comment (sunshinejr) [inline Gifski/InternetAccessPolicy.json]: I felt like the "and they can't fix what they don't know about" is a little too condescending, even though we don't have that intentions in mind. Not sure on the fix but I think we should update the sentence to be a little more nicer than this :D

Comment (sindresorhus) [reply Gifski/InternetAccessPolicy.json]: I agree. This sounds nicer.

**Comment by sindresorhus** [comment]
> any crashes that happen on your device

This makes it sound like we would receive reports about **any** crashes, even ones not related to Gifski.

File: Gifski/InternetAccessPolicy.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
...
+			"DenyConsequences" : "If you deny this connection, the developers of Gifski won't be notified of any crashes that happen on your device and thus won't be able to fix it."
```
Comment (sindresorhus) [inline Gifski/InternetAccessPolicy.json]: How about this?

```suggestion
			"DenyConsequences" : "If you deny this connection, the Gifski developers won't be notified if the app crashes and thus won't be able to fix it."
```

Comment (sunshinejr) [reply Gifski/InternetAccessPolicy.json]: ah very nice 👍

**Pull request merged**